playbooks/georchestra.yml

```diff
@@ -103,9 +103,24 @@
       publicKey: '',
     }
     tomcat_instances: {
-      proxycas: { port: 8180, control_port: 8105 },
-      georchestra: { port: 8280, control_port: 8205 },
-      geoserver: { port: 8380, control_port: 8305 }
+      proxycas: {
+        port: 8180,
+        control_port: 8105,
+        xms: 256m,
+        xmx: 512m
+      },
+      georchestra: {
+        port: 8280,
+        control_port: 8205,
+        xms: 1G,
+        xmx: 2G
+      },
+      geoserver: {
+        port: 8380,
+        control_port: 8305,
+        xms: 1G,
+        xmx: 1G
```

Comment:
XMX values very low. 
I'm not sure this will allow newer geoserver & geonetwork versions to start.
Will try it.

Reply:
GeoNetwork is currently unhappy, but unrelated at first sight

Reply:
https://github.com/georchestra/georchestra/issues/2986

Reply:
Surprised to see it still runs fine with 4Gb !